server/src/main/java/org/opensearch/common/blobstore/BlobStore.java

```diff
@@ -78,7 +78,8 @@ enum Metric {
         REQUEST_SUCCESS("request_success_total"),
         REQUEST_FAILURE("request_failures_total"),
         REQUEST_LATENCY("request_time_in_millis"),
-        RETRY_COUNT("request_retry_count_total");
+        RETRY_COUNT("request_retry_count_total"),
+        GENERIC_STATS("generic_stats");
```

Comment:
do we need it here ? We can leave it to specific blob stores to populate whatever they need in extended stats .

Reply:
Didn't get this. Both stats and extendedStats require a `Metric` type so we need a new Metric type. I didn't get what you mean by ` leave it to specific blob stores ` and didn't get how you are thinking of implementation. I kept the term generic so that respective repositories can expose their own stats.